app/scripts/views/settings.js

```diff
@@ -148,6 +149,34 @@ function ($, modal, Cocktail, Session, BaseView, AvatarMixin,
       return self._showAvatar();
     },
 
+    beforeDestroy: function () {
+      $('body.settings').fadeOut(this.FADE_OUT_SETTINGS_MS, function (){
+        $('body').removeClass('settings').show();
+      });
+    },
```

Comment:
Ugh, I didn't notice in https://github.com/mozilla/fxa-content-server/commit/207dfd69db6d34d882e3e1744d7d36aa42a42b83 that we removed the smooth transition.

/cc @shane-tomlinson